lib/contracts/builtin_contracts.rb

```diff
@@ -345,6 +345,77 @@ def validate_hash(hash)
     end
   end
 
+  # Use this for specifying contracts for keyword arguments
+  # Example: <tt>OptHash[ e: Range, f: Opt[Num] ]</tt>
+  class OptHash < CallableClass
+    def initialize(options)
+      @options = options
+    end
+
+    def valid?(hash)
+      options.all? do |key, contract|
+        Opt._valid?(hash, key, contract)
+      end
+    end
+
+    def to_s
+      "OptHash[#{options}]".gsub(/Contracts::/, "")
```

Comment:
You shouldn't need to strip the `Contracts::` prefix, the formatter should do that for you automatically: https://github.com/egonSchiele/contracts.ruby/blob/master/lib/contracts/formatters.rb#L105

Is that not working?